src/components/piece/piece.js

```diff
@@ -0,0 +1,32 @@
+import React, { useState } from 'react';
+import { useCoordinate } from './useCoordinate';
+
+const Piece = () => {
+
+  const {coordinate, movement} = useCoordinate(); 
+
+  // const [coordinate, setCoordinate] = useState({cx: 75/2 , cy: 75/2, counter: 1});
```

Comment:
never leave commented code sitting around. if you want to refer to it later, you can use git history to do so.